composer.json

```diff
@@ -13,25 +13,29 @@
 		}
 	],
 	"require": {
-		"php": ">=5.4",
+		"php": "^7.1",
 		"composer/installers": "*",
 		"kohana/core": "^3.3.0",
 		"kohana/database": "^3.3.0",
 		"openbuildings/flex-storage": "^0.1.3",
 		"kohana/image": "^3.3.0"
 	},
 	"require-dev": {
-		"phpunit/phpunit-dom-assertions": "^0.1",
-		"symfony/dom-crawler": ">=2.3",
-		"symfony/css-selector": ">=2.3",
 		"openbuildings/kohana-test-bootstrap": "^0.2",
-		"phpunit/phpunit": "~4.0.0",
-		"openbuildings/jam-monetary": "^0.1"
+		"openbuildings/jam-monetary": "^0.1",
```

Comment:
Shouldn't this package be updated with a newer version?

Reply:
Yes, updated.